x-pack/plugins/cloud_security_posture/common/types.ts

```diff
@@ -9,25 +9,31 @@ export type Evaluation = 'passed' | 'failed' | 'NA';
 /** number between 1-100 */
 export type Score = number;
 
-export interface FindingsResults {
+export interface FindingsEvaluations {
   totalFindings: number;
   totalPassed: number;
   totalFailed: number;
 }
 
-export interface Stats extends FindingsResults {
+export interface Stats extends FindingsEvaluations {
   postureScore: Score;
 }
 
-export interface ResourceTypeAgg extends FindingsResults {
-  resourceType: string;
+export interface ResourceType extends FindingsEvaluations {
```

Comment:
```suggestion
export interface Resource extends FindingsEvaluations {
```

consider renaming to `Resource` . IMO `Type` doesn't adds more information

Reply:
`Resource` would contain fields like `name`, `type`, `location` etc.
`ResourceType` is a grouping by `resource.type`, for this case it contains accumulated data for `findings evaluations` and the `resouce.type` it self.

This field will get changed in the near future since the latest product decisions are to use `rule.section` instead of `resource.type` for the grouped values.

Reply:
i don't see we already have a `Resource` type, so not sure i follow why `type` can't be a property on it and instead has to be in the name itself.

Reply:
we dont have `Resource` i just explained why `Resource` is not a fitting name for "findings grouped by resource.type field".. it would be a good name for "findings grouped by resource" which doesn't really make sense since `resource` is an object.